src/precice/Participant.cpp

```diff
@@ -132,96 +125,96 @@ int Participant::getMeshVertexSize(
 
 void Participant::setMeshVertices(
     ::precice::string_view        meshName,
-    ::precice::span<const double> positions,
+    ::precice::span<const double> coordinates,
     ::precice::span<VertexID>     ids)
 {
-  _impl->setMeshVertices(toSV(meshName), positions, ids);
+  _impl->setMeshVertices(toSV(meshName), coordinates, ids);
 }
 
 void Participant::setMeshEdge(
     ::precice::string_view meshName,
-    int                    firstVertexID,
-    int                    secondVertexID)
+    VertexID               firstVertexID,
+    VertexID               secondVertexID)
 {
   _impl->setMeshEdge(toSV(meshName), firstVertexID, secondVertexID);
 }
 
 void Participant::setMeshEdges(
     ::precice::string_view          meshName,
-    ::precice::span<const VertexID> vertices)
+    ::precice::span<const VertexID> ids)
 {
-  _impl->setMeshEdges(toSV(meshName), vertices);
+  _impl->setMeshEdges(toSV(meshName), ids);
 }
 
 void Participant::setMeshTriangle(
     ::precice::string_view meshName,
-    int                    firstVertexID,
-    int                    secondVertexID,
-    int                    thirdVertexID)
+    VertexID               firstVertexID,
+    VertexID               secondVertexID,
+    VertexID               thirdVertexID)
```

Comment:
Should we keep the parameter names this way?
Looks very redundant.

Reply:
I t

Reply:
`first`, `second`, `third` sounds good, especially since we don't want to have much camelCase in the API.

It is indeed redundant now that the type is `VertexID` and having a short arguments list is usually easier to read. I don't think we lose any value by dropping the `...VertexID`.